src/MOIWrapper.jl

```diff
@@ -70,22 +70,13 @@ end
 
 function MOI.empty!(model::Optimizer)
     old_model = model.inner
-
-    # Create new Clp object
     model.inner = Clp.ClpModel()
-
     # Copy parameters from old model into new model
-    for (option, (getter, setter)) in CLP_OPTION_MAP
-        value = getter(old_model)
-        setter(model.inner, value)
+    for (_, (getter, setter)) in CLP_OPTION_MAP
+        setter(model.inner, getter(old_model))
     end
-
     model.optimize_called = false
-
-    # Free old Clp object
-    Clp.ClpCInterface.delete_model(old_model)
```

Comment:
Yes this could possibly end up calling the finalizer twice.

Reply:
So this shouldn't have been there, but shouldn't have been the issue because of
https://github.com/JuliaOpt/Clp.jl/blob/c6121a52c6c980afa231d88944b71d90884494c0/src/ClpCInterface.jl#L230-L237
There was plausibly a weird race condition where `delete_model` got called, the function exited, the GC got run, and then `old_model` was finalized, but that doesn't seem to be the issue here.

Reply:
Yes, that was what I was thinking. CI environments will force GC to be more aggressive due to lesser memory overall. Ideally, we should just let GC clear the object and the finalizer do its job and never free it through other paths. Does that help get rid of the crashes?